<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartView</class>
 <widget class="QWidget" name="SmartView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>668</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>View</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>367</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="expressionWindow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	qproperty-alignment: 'AlignVCenter | AlignLeft';
	border-bottom: 1px solid gray;
	padding-right: 10px;
}

background-color: white;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>cos()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_atan">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>50</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>atan()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_lbracket">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>50</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rbracket">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equal">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>50</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>sin()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>90</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>sqrt()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>90</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>90</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>tan()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ln">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>130</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>ln()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mult">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>130</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>130</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_acos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>acos()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>170</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>log()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>170</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>170</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_asin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>asin()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mod">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>210</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_power">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>210</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_undo">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>210</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>UNDO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_x">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xValue">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>253</y>
      <width>187</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	qproperty-alignment: 'AlignVCenter | AlignLeft';
	border-bottom: 1px solid gray;
	padding-right: 10px;
}

background-color: white;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_PLOT">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 700 16pt &quot;Pastelova Demo&quot;;
	background-color: rgb(224, 170, 255);
	color: rgb(94, 92, 100);
}

 QPushButton:hover {
	background-color: rgb(226, 179, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>PLOT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>44</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>max x:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>min x:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>43</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>max y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>330</y>
      <width>40</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>min y:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxX">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>302</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="minX">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>332</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>-10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxY">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>302</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="minY">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>332</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>-10</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="customPlot" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>391</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
